include/nbl/video/surface/ISurfaceLinux.h

```diff
@@ -0,0 +1,34 @@
+#ifndef __NBL_I_SURFACE_LINUX_H_INCLUDED__
+#define __NBL_I_SURFACE_LINUX_H_INCLUDED__
+
+#include "nbl/core/compile_config.h"
+
+#ifdef _NBL_PLATFORM_LINUX_
+
+#include <X11/Xlib.h>
+
+namespace nbl {
+namespace video
+{
+
+class ISurfaceLinux
```

Comment:
dont call stuff to do with X11, Linux

Use X11 in the names

Reply:
yep I didn't realize Linux has more than one display servers

Reply:
well now you do, so rename ;)